src/sentry/api/event_search.py

```diff
@@ -1952,9 +1965,9 @@ def result_type_fn(function_arguments, parameter_values):
     return result_type_fn
 
 
-# When adding functions to this list please also update
-# static/sentry/app/utils/discover/fields.tsx so that
-# the UI builder stays in sync.
+# When updating this list, also check if the following need to be updated:
+# - convert_search_filter_to_snuba_query
+# - static/app/utils/discover/fields.tsx FIELDS (for discover column list and search box autocomplete)
```

Comment:
Can we update this comment to be the same as the one above?

Reply:
I made this different on purpose because I'm not sure if functions will be treated as tags. I guess I can check but also I think it doesn't really matter?